PubnativeLite/PubnativeLite/Banner/VWAdvertView.h

```diff
@@ -0,0 +1,63 @@
+//
+//  Copyright © 2020 PubNative. All rights reserved.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+#import <CoreLocation/CoreLocation.h>
+#import "VWAdRequest.h"
+#import "HyBidAdSize.h"
+#import "HyBidAdView.h"
+#import "VWAdSize.h"
+
+@class VWAdvertView;
+
+@protocol VWAdvertViewDelegate
+
+- (void)advertViewDidReceiveAd:(nonnull VWAdvertView *)adView;
+
+- (void)advertView:(nonnull VWAdvertView *)adView didFailToReceiveAdWithError:(nullable NSError *)error;
+
+@end
+
+@interface VWAdvertView : UIView<HyBidAdViewDelegate>
+
+@property (nonatomic, strong) HyBidAdView * _Nonnull adView;
```

Comment:
Can we move this HyBidAdView property to .m? I think it is not called by publisher from Demo App, so we can hide it. 😉